web/templates/user/edit.html.haml

```diff
@@ -1,5 +1,5 @@
-.measure-wide.center
+.measure.center
   %h2.tc Edit user
   = render "form.html", changeset: @changeset, action: user_path(@conn, :update, @user)
```

Comment:
I've never seen underscores in partials in phoenix land. Best to stick with the idioms